### PR TITLE
Crowdmap single session

### PR DESCRIPTION
**Pull request by AkkarinForest**

### CODE REVIEW

File: app/assets/stylesheets/modules/filters.css.scss [hunk 1]
```diff
@@ -41,3 +41,8 @@
   right: 0;
   top: 0;
 }
+
+.disabled-area {
+  pointer-events: none;
+  opacity: 0.4;
+}
```
Comment (AkkarinForest) [inline app/assets/stylesheets/modules/filters.css.scss]: fyi @gitjul - that's what I added, feel free to adjust accordingly.

File: app/javascript/elm/src/Main.elm [hunk 2]
```diff
@@ -922,6 +917,27 @@ viewSensorFilter sensors selectedSensorId =
...
+viewCrowdMapOptions : Bool -> Int -> Bool -> Html Msg
+viewCrowdMapOptions isCrowdMapOn crowdMapResolution isDisabled =
+    let
+        class_ =
+            case isDisabled of
+                True ->
+                    "disabled-area"
+
+                False ->
+                    ""
```
Comment (AkkarinForest) [inline app/javascript/elm/src/Main.elm]: Is there a better/shorter way to have a class dependent on some bool value?

File: app/assets/stylesheets/modules/filters.css.scss (hunk 1 above)
Comment (gitjul) [reply app/assets/stylesheets/modules/filters.css.scss]: looks ok, other disabled stuff had 0.4 opacity so it should be 👍

File: app/javascript/elm/src/Data/SelectedSession.elm [hunk 3]
```diff
@@ -169,3 +170,13 @@ view session heatMapThresholds linkIcon toMsg =
             , button [ class "button button--primary action-button action-button--copy-link", Events.onClick <| toMsg tooltipId, id tooltipId ] [ img [ src linkIcon, alt "Link icon" ] [] ]
             ]
         ]
+
+
+isSessionSelected : WebData SelectedSession -> Bool
```
Comment (3v0k4) [inline app/javascript/elm/src/Data/SelectedSession.elm]: I guess you could use `https://package.elm-lang.org/packages/krisajenkins/remotedata/latest/RemoteData#isSuccess`

File: app/javascript/elm/src/Main.elm (hunk 2 above)
Comment (3v0k4) [reply app/javascript/elm/src/Main.elm]: Yep :) https://package.elm-lang.org/packages/elm/html/latest/Html-Attributes#classList

File: app/javascript/elm/src/Main.elm [hunk 4]
```diff
@@ -922,6 +917,27 @@ viewSensorFilter sensors selectedSensorId =
         ]
 
 
+viewCrowdMapOptions : Bool -> Int -> Bool -> Html Msg
+viewCrowdMapOptions isCrowdMapOn crowdMapResolution isDisabled =
```
Comment (3v0k4) [inline app/javascript/elm/src/Main.elm]: You could pass the selectedSession in and checking here https://package.elm-lang.org/packages/krisajenkins/remotedata/latest/RemoteData#isSuccess

**Comment by 3v0k4** [comment]
Good stuff!

**Pull request merged**